manifests/microservices/runnumber-rest.yaml

```diff
@@ -7,14 +7,17 @@ spec:
   selector:
     matchLabels:
       app: runnumber-rest
-  minReadySeconds: 5
   template:
     metadata:
       labels:
         app: runnumber-rest
     spec:
+      initContainers:
+      - name: wait-for-run-postgres
+        image: busybox:1.28
+        command: ['sh','-c', 'until nslookup runnumber-rest-svc-ext.microservices.svc.cluster.local; do echo waiting for run-postgres to run; sleep 1; done']            
       containers:
-      - image: ghcr.io/dune-daq/pocket-runnumber-rest:0.8
+      - image: ghcr.io/dune-daq/pocket-runnumber-rest:1.0
```

Comment:
I'm not showing any code changes pending up at https://github.com/DUNE-DAQ/microservices to track the container tag changes...

Reply:
I'm not on the DUNE DAQ microservices group so I couldn't upload my changes on to microservices to track the container changes.

Reply:
@TiagoTAlves , can you fork it, create a PR on microservices and advertise it here?

Reply:
I'm showing https://github.com/DUNE-DAQ/microservices/tree/titavare/run-number exists now.  I pushed a few additional changes into that branch so the containers can run as non-root.

Long term I'd like to store the kubernetes yaml for the service with the code that builds the container.  That way if things change further down the line the deployments are easy to sync up with specific container identities.